x/emissions/keeper/keeper.go

```diff
@@ -3410,3 +3439,125 @@ func (k *Keeper) GetPreviousForecasterScoreRatio(ctx context.Context, topicId To
 	}
 	return forecastTau, nil
 }
+
+// AddQualifiedInferer adds an inferer to the qualified inferers set for a topic
+func (k *Keeper) AddQualifiedInferer(ctx context.Context, topicId TopicId, inferer ActorId) error {
+	if err := types.ValidateTopicId(topicId); err != nil {
```

Comment:
This also should check the MaxINferersToReward

Reply:
This is already checked in the unique function that calls `AddQualifiedInferer `